ts/Core/Pointer.ts

```diff
@@ -1830,25 +1844,36 @@ class Pointer {
      * @private
      * @function Highcharts.Pointer#scaleGroups
      */
-    public scaleGroups(attribs?: Series.PlotBoxObject, clip?: boolean): void {
+    public scaleGroups(attribs?: Series.PlotBoxObject, clip?: BBoxObject, pinchedAxes?: Axis[]): void {
 
         const chart = this.chart;
-
+        let axes = pinchedAxes || chart.axes;
         // Scale each series
-        chart.series.forEach(function (series): void {
-            const seriesAttribs = attribs || series.getPlotBox(); // #1701
-            if (series.xAxis && series.xAxis.zoomEnabled && series.group) {
-                series.group.attr(seriesAttribs);
-                if (series.markerGroup) {
-                    series.markerGroup.attr(seriesAttribs);
-                    series.markerGroup.clip(
-                        clip ? (chart.clipRect as any) : (null as any)
-                    );
-                }
-                if (series.dataLabelsGroup) {
-                    series.dataLabelsGroup.attr(seriesAttribs);
+        axes.forEach((axis: Axis): void => {
+            axis.series.forEach((series: Series): void => {
+                const seriesAttribs = attribs || series.getPlotBox(); // #1701
+                const isX = axis.isXAxis;
+
+                const attr: any = {};
+                let scale = `scale${isX ? 'X' : 'Y'}`;
+                let translate = `translate${isX ? 'X' : 'Y'}`;
+                attr[scale] = (seriesAttribs as any)[scale];
+                const transformScale = chart.inverted ? 1 / attr[scale] : attr[scale];
+                attr[translate] = (seriesAttribs as any)[translate] + (attribs ? (transformScale * axis.pos) : 0);
+
+                if (series.xAxis && series.xAxis.zoomEnabled && series.group) {
+                    series.group.attr(attr);
+                    if (series.markerGroup) {
+                        series.markerGroup.attr(attr);
+                        series.markerGroup.clip(
+                            clip ? (chart.clipRect as any) : (null as any)
```

Comment:
What if the `chart.clipRect` isn't defined? I guess that is the reason for casting to `any`.

Reply:
This was the previous solution. I only edited the moment, when it is invoked. This part of code needs heavy refactoring, and clipping will be handled soon, so I'd suggest to leave it as it is.

Reply:
Yep! Fine ;)